manifests/kustomize/base/metadata/metadata-configmap.yaml

```diff
@@ -4,7 +4,7 @@ kind: ConfigMap
 metadata:
   name: metadata-configmap
   labels:
-    component: metadata-server
```

Comment:
The first and the second configmap here are not related to grpc server and they specify configs if the components wants to directly communicate with mysql database. These were left as is for backward compatibility. I suggest we keep them as is and only change the third config-map here.

Reply:
Same applies for helm-chart configs as well.

Reply:
You are right, I fixed it.
Please confirm that it's ok

Reply:
Sorry I had not seen this comment. The change looks good to me.